feat: assign taxonomy to organizations [TEMP]

rpenido opened the pull request:
TEMP

rpenido committed:
feat: add manage-orgs modal

rpenido committed:
fix: menu layout and input clear

rpenido committed:
feat: add api

rpenido committed:
refactor: move useTaxonomyDetail hooks to avoid circular dependency

manage-orgs -> taxonomy-detail -> taxonomy-menu -> manage-orgs

rpenido committed:
fix: undo css change

rpenido committed:
fix: add confirm modal

rpenido committed:
test: add manage org menu test

rpenido committed:
Merge branch 'rpenido/fal-3532-import-taxonomy' into rpenido/fal-3543-assign-taxonomy-to-organizations

rpenido committed:
fix: merge conflicts

rpenido committed:
Merge branch 'rpenido/fal-3532-import-taxonomy' into rpenido/fal-3543-assign-taxonomy-to-organizations

rpenido committed:
fix: disable element if is allOrg

rpenido committed:
test: add tests

ChrisChV commented on src/taxonomy/manage-orgs/ManageOrgsModal.scss:
```diff
@@ -0,0 +1,7 @@
+  overflow: visible !important;
+
+  .pgn__modal-body {
+    overflow: visible;
```
@rpenido Could you add the reason of this style in a comment?

rpenido replied:
Done [9fbe76e](https://github.com/open-craft/frontend-app-course-authoring/pull/23/commits/9fbe76ebefb4ec86d57b0fce3d171f5bb84cddaa)!

Also updated the spacing here: [aca5ce5](https://github.com/open-craft/frontend-app-course-authoring/pull/23/commits/aca5ce5873580c50817e9404b27de54bfc71a744)

![image](https://github.com/open-craft/frontend-app-course-authoring/assets/849463/fbcebe3e-6887-44e2-92b2-aee7077f06ab)

ChrisChV reviewed (commented):
@rpenido I think you need to add some padding here. Like the modal on [Figma](https://www.figma.com/file/DmUC2e7BAoQsGDq4KgApPa/Content-Tagging-MVP?type=design&node-id=9715-51696&mode=design&t=RQE5B5zPri3Ez4TU-4)

![photo_2023-12-20_08-34-10](https://github.com/open-craft/frontend-app-course-authoring/assets/11827305/2619db10-0317-4fea-ad17-e48b5644f60b)

rpenido committed:
style: add comment about modal style to allow dropdown overflow

rpenido committed:
fix: add spacing

rpenido committed:
fix: dont pass null children

rpenido committed:
fix: cleaning code

rpenido committed:
test: add tests

rpenido committed:
test: add more tests

rpenido committed:
fix: typo

ChrisChV commented:
@rpenido Looks good, great work! :+1:
- [X] I tested this: I tested all the workflow.
- [X] I read through the code and considered the security, stability and performance implications of the changes.
- [X] I tested that the UI can be used with a keyboard only (tab order, keyboard controls).
- [X] Includes tests for bugfixes and/or features added.
- [X] Includes documentation

renovate[bot] committed:
fix(deps): update dependency @edx/frontend-lib-content-components to v1.177.9 (#762)

Co-authored-by: renovate[bot] <29139614+renovate[bot]@users.noreply.github.com>

yusuf-musleh committed:
feat: Add filter taxonomies by org (#755)

This implements filtering taxonomies on the taxonomy list page by selecting organization name, all taxonomies, or unassigned taxonomies.

KristinAoki committed:
fix: uploading progress percentage (#763)

ArturGaspar committed:
feat: error page on invalid course key (#761)

ayesha-waris committed:
chore: removed reported_content_email_notifications_flag dependency (#775)

rpenido committed:
feat: add import taxonomy feature [FC-0036] (#675)

This change adds a new button in the Taxonomy List to allow users to create new taxonomies by importing a CSV/JSON file.

yusuf-musleh committed:
feat: Add v2 lib components content tags support (#771)

Add support for fetching content data for Library V2 components in content tags drawer.

rpenido committed:
Merge branch 'master' into rpenido/fal-3543-assign-taxonomy-to-organizations

rpenido committed:
test: fix test

rpenido commented:
Closed in favor of https://github.com/openedx/frontend-app-course-authoring/pull/732
